File: inst/examples/readGZ.R
# dyn.load(system.file("libs", "RSPerl.so", package = "RSPerl"), local = FALSE)

library(RSPerl)
.PerlInit()

.PerlPackage(c("IO::File", "IO::Uncompress::AnyUncompress"))
f = "/home/duncan/Projects/org/omegahat/Interfaces/Perl/tests/test.gz"

k =.PerlNew("IO::Uncompress::AnyUncompress", f)

File: inst/examples/readGZ.pl
#!/usr/bin/perl -W

use IO::File;
use warnings ;
use IO::Uncompress::AnyUncompress qw(anyuncompress $AnyUncompressError) ;

my $file = "test.gz";
my $fh;

$fh = new IO::Uncompress::AnyUncompress $file
	    or die "Cannot get the uncompressed version";


my $z;
$z = $fh->getline();
print $z, "\n";
$z = <$fh>;
print $z, "\n";

